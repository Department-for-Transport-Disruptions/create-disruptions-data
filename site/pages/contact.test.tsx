import renderer from "react-test-renderer";
import { describe, expect, it } from "vitest";
import Contact from "./contact.page";

describe("contact", () => {
    it("should render correctly", () => {
        const tree = renderer.create(<Contact />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
