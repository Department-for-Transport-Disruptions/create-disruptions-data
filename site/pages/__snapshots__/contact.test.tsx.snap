// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`contact > should render correctly 1`] = `
[
  <div
    className="govuk-phase-banner max-w-[960px] w-[90%] border-0 mx-auto app-phase-banner app-width-container"
  >
    <p
      className="govuk-phase-banner__content"
    >
      <strong
        className="govuk-tag govuk-phase-banner__content__tag"
      >
        Beta
      </strong>
      <span
        className="govuk-phase-banner__text"
      >
        This is a new service. Help us improve it and
         
        <a
          className="govuk-link"
          href="/feedback"
          id="feedback-link"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          give your feedback (opens in new tab)
        </a>
        .
      </span>
    </p>
  </div>,
  <div
    className="govuk-width-container"
  >
    <button
      className="govuk-back-link mb-7"
      onClick={[Function]}
      type="button"
    >
      Back
    </button>
    <main
      className="govuk-main-wrapper"
      id="main-content"
    >
      <div
        className="govuk-grid-row"
      >
        <div
          className="govuk-grid-column-two-thirds"
        >
          <h1
            className="govuk-heading-l"
          >
            Contact the Create Transport Disruption Data Service team
          </h1>
          <h2
            className="govuk-heading-m"
          >
            Feedback and support
          </h2>
          <p
            className="govuk-body"
          >
            If you are experiencing technical issues, or if you have any suggestions, comments or criticisms, please contact the Create Transport Disruption Data team through one of the channels below.
          </p>
          <p
            className="govuk-body"
          >
            The Help Desk is available Monday to Friday, 9am to 5pm (excluding Bank Holidays in England and Wales, and the 24th of December).
          </p>
          <p
            className="govuk-body"
          >
            The Help Desk can be contacted by telephone or email as follows.
          </p>
          <p
            className="govuk-body"
          >
            Telephone: 
            0800 028 0930
            <br />
            Email:
             
            <a
              className="govuk-link"
              href="mailto:bodshelpdesk@kpmg.co.uk"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              bodshelpdesk@kpmg.co.uk
            </a>
          </p>
          <h3
            className="govuk-heading-s"
          >
            Related services
          </h3>
          <p
            className="govuk-body"
          >
            <a
              aria-label="go to the bus open data service"
              className="underline govuk-link"
              href="https://www.bus-data.dft.gov.uk/contact/"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              https://www.bus-data.dft.gov.uk/contact/
            </a>
            <br />
            <br />
            The Bus Open Data Service deals with queries relating to the use of Bus Open Data.
          </p>
        </div>
        <div
          className="govuk-grid-column-one-third"
        >
          <h2
            className="govuk-heading-s"
          >
            Create Transport Disruption Data Service
          </h2>
          <p
            className="govuk-body"
          >
            The Create Transport Disruption Data Service gives local transport authorities and operators within England the ability to create disruptions for bus, light rail, ferry and tram services Help documents
          </p>
        </div>
      </div>
    </main>
  </div>,
  <footer
    className="govuk-footer"
    role="contentinfo"
  >
    <div
      className="govuk-width-container"
    >
      <div
        className="govuk-footer__meta"
      >
        <div
          className="govuk-footer__meta-item govuk-footer__meta-item--grow"
        >
          <h2
            className="govuk-visually-hidden"
          >
            Support links
          </h2>
          <ul
            className="govuk-footer__inline-list"
          >
            <li
              className="govuk-footer__inline-list-item"
            >
              <a
                className="govuk-footer__link"
                href="/contact"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
              >
                Help
              </a>
            </li>
            <li
              className="govuk-footer__inline-list-item"
            >
              <a
                className="govuk-footer__link"
                href="/cookies"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
              >
                Cookies
              </a>
            </li>
            <li
              className="govuk-footer__inline-list-item"
            >
              <a
                className="govuk-footer__link"
                href="/contact"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
              >
                Contact
              </a>
            </li>
            <li
              className="govuk-footer__inline-list-item"
            >
              <a
                className="govuk-footer__link"
                href="/accessibility"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
              >
                Accessibility
              </a>
            </li>
            <li
              className="govuk-footer__inline-list-item"
            >
              <a
                className="govuk-footer__link"
                href="https://www.gov.uk/help/terms-conditions"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
              >
                Terms and conditions
              </a>
            </li>
            <li
              className="govuk-footer__inline-list-item"
            >
              <a
                className="govuk-footer__link"
                href="/privacy"
                onClick={[Function]}
                onMouseEnter={[Function]}
                onTouchStart={[Function]}
              >
                Privacy
              </a>
            </li>
          </ul>
          <p
            className="govuk-footer__body"
          >
            Built by the
             
            <a
              className="govuk-footer__link"
              href="https://www.gov.uk/government/organisations/department-for-transport"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
              rel="noreferrer"
              target="_blank"
            >
              Department for Transport
            </a>
          </p>
          <svg
            aria-hidden="true"
            className="govuk-footer__licence-logo"
            focusable="false"
            height="17"
            viewBox="0 0 483.2 195.7"
            width="41"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M421.5 142.8V.1l-50.7 32.3v161.1h112.4v-50.7zm-122.3-9.6A47.12 47.12 0 0 1 221 97.8c0-26 21.1-47.1 47.1-47.1 16.7 0 31.4 8.7 39.7 21.8l42.7-27.2A97.63 97.63 0 0 0 268.1 0c-36.5 0-68.3 20.1-85.1 49.7A98 98 0 0 0 97.8 0C43.9 0 0 43.9 0 97.8s43.9 97.8 97.8 97.8c36.5 0 68.3-20.1 85.1-49.7a97.76 97.76 0 0 0 149.6 25.4l19.4 22.2h3v-87.8h-80l24.3 27.5zM97.8 145c-26 0-47.1-21.1-47.1-47.1s21.1-47.1 47.1-47.1 47.2 21 47.2 47S123.8 145 97.8 145"
              fill="currentColor"
            />
          </svg>
          <span
            className="govuk-footer__licence-description"
          >
            All content is available under the
             
            <a
              className="govuk-footer__link"
              href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
              rel="license"
            >
              Open Government Licence v3.0
            </a>
            , except where otherwise stated
          </span>
        </div>
        <div
          className="govuk-footer__meta-item"
        >
          <a
            className="govuk-footer__link govuk-footer__copyright-logo bg-govuk-crest"
            href="https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/"
            onClick={[Function]}
            onMouseEnter={[Function]}
            onTouchStart={[Function]}
          >
            Â© Crown copyright
          </a>
        </div>
      </div>
    </div>
  </footer>,
]
`;
